extensions/mssql/src/objectManagement/ui/serverRoleDialog.ts

```diff
@@ -27,22 +28,24 @@ export class ServerRoleDialog extends ObjectManagementDialogBase<ObjectManagemen
 
 
 	constructor(objectManagementService: IObjectManagementService, options: ObjectManagementDialogOptions) {
-		super(objectManagementService, options);
+		super(objectManagementService, options, false, false);
```

Comment:
just found db role and app role also don't support effective permission

Reply:
fixed.